scripts/html_lint.py

```diff
@@ -106,11 +108,21 @@ def main():
         return 1
 
     exclude = [_DISABLE_MAP[d] for d in disable if d in _DISABLE_MAP]
-    clean_html = template_remover.clean(io.open(options['FILENAME']).read())
-    results = html_linter.lint(clean_html, exclude=exclude)
+
+    path = options['FILENAME']
+    is_file = os.path.isfile(path)
+    html_files = [path] if is_file else glob.glob('{}/*.html'.format(path))
+
+    results = []
+    for html_file in html_files:
+        clean_html = template_remover.clean(io.open(html_file).read())
+        result = html_linter.lint(clean_html, exclude=exclude)
+        if result:
+            results.append(result)
```

Comment:
I would like to see the results as seen as they are produced. This is specially important if the directory has many files.